### ISSUE TITLE
Create 3.6 branch

### ISSUE THREAD

**Issue by Eyjafjallajokull** [pybee/Python-macOS-template#1]
```console
python setup.py macos
running macos
 * Writing application template...
Project template: https://github.com/pybee/Python-macOS-template.git
Traceback (most recent call last):
  File "/Users/polejniczak/src/slick/venv/lib/python3.6/site-packages/cookiecutter/vcs.py", line 129, in clone
    stderr=subprocess.STDOUT,
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 418, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['git', 'checkout', '3.6']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "setup.py", line 79, in <module>
    'toga-django',
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/Users/polejniczak/src/slick/venv/lib/python3.6/site-packages/briefcase/app.py", line 336, in run
    self.generate_app_template()
  File "/Users/polejniczak/src/slick/venv/lib/python3.6/site-packages/briefcase/app.py", line 195, in generate_app_template
    'secret_key': self.secret_key,
  File "/Users/polejniczak/src/slick/venv/lib/python3.6/site-packages/cookiecutter/main.py", line 63, in cookiecutter
    no_input=no_input,
  File "/Users/polejniczak/src/slick/venv/lib/python3.6/site-packages/cookiecutter/repository.py", line 84, in determine_repo_dir
    no_input=no_input,
  File "/Users/polejniczak/src/slick/venv/lib/python3.6/site-packages/cookiecutter/vcs.py", line 132, in clone
    if 'not found' in clone_error.output.lower():
TypeError: a bytes-like object is required, not 'str'
```

**Comment by dawranliou** [comment]
Hey @freakboy3742 could you create a new branch for Python 3.6? I think this has to be done by the person who has push privilege. Thanks.

**Comment by freakboy3742** [comment]
@dawran6 The problem is that while it's easy enough to create a 3.6 branch for this template, there's no support package to back it up. 

I've been working on fixing this, using a patch from a branch by @dimaqq  - however, it's not building because of the problem I described [here] (https://gist.github.com/freakboy3742/2cdb1c68c189539d588ef766432b1d97). This problem is also affecting the release of new 3.4 and 3.5 builds. If anyone can solve the problem, I'll gladly push a new release, as well as a new template branch.

**Comment by dawranliou** [comment]
Oh I see! I was looking for contributions that I can make and found the same issue. Thought the fix was as simple as to create a new branch. For now I'll switch to 3.5. Thanks for the timely response.

**Comment by tmontes** [comment]
Given that #3 was merged against the 3.6 branch, I suppose this issue can be closed, correct? :)

**Comment by freakboy3742** [comment]
I guess so!
